Makefile

```diff
@@ -58,7 +58,10 @@ test :
 
 .PHONY : test-with-cov
 test-with-cov :
-	pytest --color=yes -rf --cov-config=.coveragerc --cov=$(SRC) --durations=40 -k "not sniff_test" $(SRC) scripts/
+	pytest --color=yes -rf --durations=40 -k "not sniff_test" \
```

Comment:
I don't think we have sniff tests in here anymore, so that argument can probably be removed.